src/bindings/python/tests/test_graph/test_preprocess.py

```diff
@@ -728,3 +728,52 @@ def test_graph_set_layout_by_layout_class_thow_exception():
         layout = Layout("1-2-3D")
         ppp.input().model().set_layout(layout)
     assert "Layout name is invalid" in str(e.value)
+
+
+@pytest.mark.parametrize(
+    ("pads_begin", "pads_end", "values", "mode"),
+    [([0, 0, 0, 0], [0, 0, 1, 1], 0, PaddingMode.CONSTANT)])
```

Comment:
Overall LGTM, could we please get a few more test cases? Just 2 or 3 would be enough. Also please check the expected ops, return type and operator count, you can use the other tests as inspiration (for example `def test_graph_preprocess_model()`).